portal-kernel/src/com/liferay/portal/kernel/templateparser/TemplateNode.java

```diff
@@ -209,13 +208,9 @@ public String getUrl() {
 
 		String data = (String)get("data");
 
-		JSONObject jsonObject = JSONUtil.createJSONObject(data);
-
-		if (jsonObject == null) {
-			return StringPool.BLANK;
-		}
-
 		try {
+			JSONObject jsonObject = JSONFactoryUtil.createJSONObject(data);
```

Comment:
@slnn I did not include this change, as this is going to generate an exception when jsonObject is null, it is better that we keep the original checking.